docs/guidance/publish/system_architectures.md

```diff
@@ -1,102 +1,104 @@
+This guidance describes some design approaches with their advantages and disadvantages. This is not an extensive list, but it can be used to inform the design of the publication system.
 
 ## On-demand transformation from live systems
 
-In this scenario each originating system converts data directly in OCDS format on demand. Data is not stored according to OCDS but is converted each time the conversion process is invoked. This can be the easiest path when all contracting processes are managed by a single live system, although it requires modifications to the original system to add an OCDS conversion module.
+In this scenario, each source system converts data in OCDS format on demand. Data is not stored but gets converted each time an user o third party invokes the conversion process. This is the easiest path when a single live system manages all contracting processes. But it requires modifications to the system to add an OCDS conversion module.
 
-One approach in this scenario is to write to a web-accessible file system, as illustrated below, and update it periodically by invoking the conversion module.
-
-![Direct Publication 1](../../_static/png/directPublication1.png)
-
-Another one is to implement an API, which performs data transformation on the fly, each time a request is received.
+An API performs data transformation on the fly each time it receives a request.
 
 ![Direct Publication 2](../../_static/png/directPublication2.png)
 
-In both alternatives, the OCDS Conversion module produces OCDS releases and/or records, wrapped in release/record packages.
+The conversion module produces OCDS releases and/or records wrapped in packages.
 
-The file system approach ensures that each OCDS document can be accessed through a persistent URL, but the volume of data can grow quickly as plain files can take significant space. Bulk downloads can be generated periodically and stored in the file system.
+This approach does not need extra storage space. But it may be not possible to provide persistent URLs for releases, nor a change history for each process.
 
-The API approach does not require additional storage space, but it may be not possible to provide persistent URLs for individual releases. Bulk downloads can be provided as part of the API, although it may stress the live systems with large data queries.
+We can think of releases as snapshots of a contracting process in a given time. To be able to rebuild an snapshot for the past depends on how much historical data the source system saves. Also, we need to determine how to identify a snapshot, e.g. if a version id is available in the source system. The Easy releases guidance explains how to achieve a conformant OCDS implementation where it is not possible to provide a full change history.
```

Comment:
I think it's better to leave the discussion of releases and records to the other docs pages and just reference them from here.

```suggestion
The easy releases guidance explains how to achieve a conformant OCDS implementation where it is not possible to provide a full change history.
```